component/org.wso2.carbon.identity.dpop/src/main/java/org/wso2/carbon/identity/dpop/token/binder/DPoPBasedTokenBinder.java

```diff
@@ -202,14 +202,14 @@ private String retrieveTokenBindingValueFromDPoPHeader(HttpServletRequest reques
     private boolean validateDPoPHeader(Object request, TokenBinding tokenBinding) throws IdentityOAuth2Exception,
             ParseException {
 
-        if (!((HttpServletRequest) request).getHeader(DPoPConstants.AUTHORIZATION_HEADER)
-                .startsWith(DPoPConstants.OAUTH_DPOP_HEADER)) {
+        if (!((HttpServletRequest) request).getRequestURI().equals(("/oauth2/revoke")) &&
```

Comment:
Shall we use a constant for "/oauth2/revoke"?

Reply:
Fixed